src/status_im/ui/screens/notifications_center/styles.cljs

```diff
@@ -37,6 +42,20 @@
    :padding-right   5
    :flex-direction  :row})
 
+(def reply-message-container
+  {:height          22
+   :align-self      :baseline
+   :align-items :center
+   :border-radius   11
+   :border-color    colors/gray-transparent-40
+   :border-width    1
+   :margin-top      6
+   :margin-bottom   10
+   :margin-right    15
+   :padding-left    5
```

Comment:
The odd numbers look odd. Have you looked at `spacing` namespace?

Reply:
I'll take a look at it!